include/mapbox/cheap_ruler.hpp

```diff
@@ -348,6 +340,9 @@ class CheapRuler {
 private:
     double ky;
     double kx;
+    static double long_diff(double a, double b) {
+        return std::remainder(a - b, 360);
+    }
```

Comment:
Nitpick: can we rename this to `longDiff` for consistency with other method names?

Reply:
Done